gladier_tools/posix/asym_decrypt.py

```diff
@@ -0,0 +1,74 @@
+from gladier import GladierBaseTool, generate_flow_definition
+
+
+def asym_decrypt(**data):
+    import os
+    from cryptography.hazmat.primitives.serialization import \
+        load_ssh_private_key
+    from cryptography.hazmat.primitives.asymmetric import padding
+    from cryptography.hazmat.primitives import hashes
+
+    private_file = data.get('private_key_path', '~/.ssh/id_rsa')
```

Comment:
I think this would be better passed in as `flow_input`, that way the runner of the flow can double check the path in the Flows Webapp

Reply:
`private_key_path` is passed in as flow_input... not sure i understand what you mean

Reply:
Answered offline, but posting the docs here in case anyone is following here: https://gladier.readthedocs.io/en/latest/gladier_tools.html